Content.Shared/Roles/JobPrototype.cs

```diff
@@ -53,6 +53,13 @@ public sealed partial class JobPrototype : IPrototype
         [DataField("setPreference")]
         public bool SetPreference { get; private set; } = true;
 
+        /// <summary>
+        ///     Whether this job should show in the ID Card Console.
+        ///     If set to null, it will default to SetPreference's value.
+        /// </summary>
+        [DataField("overrideConsoleVisibility")]
+        public bool? OverrideConsoleVisibility { get; private set; } = null;
+
```

Comment:
```suggestion
        /// <summary>
        ///     Whether this job should show in the ID Card Console.
        ///     If set to null, it will default to SetPreference's value.
        /// </summary>
        [DataField]
        public bool? OverrideConsoleVisibility { get; private set; } = null;

```